docs/tutorials/11_GMPHDTutorial.py

```diff
@@ -0,0 +1,577 @@
+    # state from a live track, we add the state to that track. Otherwise, we generate a new track if the reduced
+    # state's weight is high enough (ie we are sufficiently certain that it is a new track).
+    for reduced_state in reduced_states:
+        # Add the reduced state to the list of Gaussians that we will plot later. Have a low threshold to eliminate some
+        # clutter that would make the graph busy and hard to understand
+        if reduced_state.weight > 0.05: all_gaussians[n].append(reduced_state)
+
+        tag = reduced_state.tag
+        # Here we check to see if the state has a sufficiently high weight to consider being added.
+        if reduced_state.weight > state_threshold:
+            # Check if the reduced state belongs to a live track
+            for track in tracks:
+                track_tags = [state.tag for state in track.states]
+
+                if tag in track_tags:
+                    track.append(reduced_state)
+                    tracks_by_time[n].append(reduced_state)
+                    break
+            else:  # Execute if no "break" is hit; i.e. no track with matching tag
+                # Make a new track out of the reduced state
+                new_track = Track(reduced_state)
+                tracks.add(new_track)
+                tracks_by_time[n].append(reduced_state)
+
+
+# %%
+# ### Plot the Tracks
+# First, determine the x and y range for axes. We want to zoom in as much as possible
+# on the measurements and tracks while not losing any of the information. This section
+# is not strictly necessary as we already set the field of view to be the range [-100, 100]
+# for both x and y. However, sometimes an object may leave the field of view. If you want
+# to ignore objects that have left the field of view, comment out this section and define
+# the variables
+# x_min = y_min = -100 and x_max = y_max = 100.
+
+x_min, x_max, y_min, y_max = 0, 0, 0, 0
+
+# Get bounds from the tracks
+for track in tracks:
+    for state in track:
+        x_min = min([state.state_vector[0], x_min])
+        x_max = max([state.state_vector[0], x_max])
+        y_min = min([state.state_vector[2], y_min])
+        y_max = max([state.state_vector[2], y_max])
+
+# Get bounds from measurements
+for measurement_set in all_measurements:
+    for measurement in measurement_set:
+        if type(measurement) == TrueDetection:
+            x_min = min([measurement.state_vector[0], x_min])
+            x_max = max([measurement.state_vector[0], x_max])
+            y_min = min([measurement.state_vector[1], y_min])
+            y_max = max([measurement.state_vector[1], y_max])
+
+# Plot the tracks
+plotter = Plotter()
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.plot_measurements(all_measurements, [0, 2], color='g')
+plotter.plot_tracks(tracks, [0, 2], uncertainty=True)
+plotter.ax.set_xlim(x_min-5, x_max+5)
+plotter.ax.set_ylim(y_min-5, y_max+5)
+
+#plotter.fig
```

Comment:
```suggestion
```
I assume we don't need this as it's commented out.

Reply:
Yes, sorry about that!